projects/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
 
-# Create your models here.
+from companies.models import CompanyProfile
+
+
+PROJECT_STATUSES = [
+    ("active", "Active"),
+    ("completed", "Completed"),
+]
+
+class Project(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    status = models.CharField(choices=PROJECT_STATUSES, blank=False)
+    information = models.TextField()
+    required_funding = models.DecimalField(max_digits=15, decimal_places=2, blank=False, null=False)
+    raised_amount = models.DecimalField(max_digits=15, decimal_places=2, blank=True, null=True)
+    company = models.ForeignKey(CompanyProfile, on_delete=models.CASCADE)
```

Comment:
`related_name` would help if you want to do something like `company.projects.all()` in the future.
```
company = models.ForeignKey(CompanyProfile, on_delete=models.CASCADE, related_name="projects")
```